src/components/Comments.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react'
+import { getCommentsFromArticle } from '../../utils'
+
+export default function Comments(id) {
+	const [comments, setComments] = useState([])
+	const [isLoading, setIsLoading] = useState(true)
+
+	useEffect(() => {
+		getCommentsFromArticle(id).then(({ comments }) => {
+			setComments(comments)
+			setIsLoading(false)
+		})
+	}, [])
+
+    console.log(comments)
+
+	if (isLoading) return <p>Loading Comments Section... wait patiently </p>
+
+    return (
+        <section className="Comments">
```

Comment:
Please display a message to the user in case there are no comments for the article. Something like: 'No comments yet'.